vhdbuilder/packer/install-dependencies.sh

```diff
@@ -110,6 +110,9 @@ fi
 downloadContainerdWasmShims
 echo "  - containerd-wasm-shims ${CONTAINERD_WASM_VERSIONS}" >> ${VHD_LOGS_FILEPATH}
 
+downloadSecureTLSBootstrapExecPlugin
```

Comment:
assuming we'd mainly want this installed on the VHD already so adding here as well